google/api_core/grpc_helpers.py

```diff
@@ -288,6 +300,7 @@ def create_channel(
     default_scopes=None,
     default_host=None,
     compression=None,
+    attempt_direct_path: Optional[bool] = None,
```

Comment:
```suggestion
    attempt_direct_path: Optional[bool] = False,
```

Since it's meant to be a Boolean, might as well.

Reply:
Fixed in https://github.com/googleapis/python-api-core/pull/583/commits/0f3a0e4c4e8afa02c3733eb246d548609c47d730